### PR TITLE
About page

### PR DESCRIPTION
**Pull request by Fetii1**

### CODE REVIEW

File: components/sections/FooterSection.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import React from 'react'
+
+export default function FooterSection() {
+  return (
+    <div>
```
Comment (Afan417) [inline components/sections/FooterSection.js:5]: remove extra div, line 5 and 9

File: components/sections/about/AboutText.js [hunk 2]
```diff
@@ -0,0 +1,68 @@
+import Image from 'next/image'
+import React from 'react'
+
+export default function AboutText() {
```
Comment (Afan417) [inline components/sections/about/AboutText.js:4]: you can move this inside ``AboutSection.js``

File: components/sections/about/HeaderSection.js [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+export default function HeaderSection() {
+  return (
+    <div
+      className="w-full h-32 bg-red-950 text-4xl items-center flex justify-start"
+      style={{ backgroundImage: `url(/img/redLines.png)` }}
+    >
+      <h1 className="text-white font-bold pl-16">Rreth Nesh</h1>
+    </div>
+  )
+}
```
Comment (Afan417) [inline components/sections/about/HeaderSection.js:12]: Move this into ``components`` folder not sections
and make title dynamic as prop

      ``<h1 className="text-white font-bold pl-16">{title}</h1>``

File: pages/about.js [hunk 4]
```diff
@@ -1,9 +1,12 @@
+import AboutText from '@/components/sections/about/AboutText'
+import HeaderSection from '@/components/sections/about/HeaderSection'
 import Layout from '@/layout/Layout'
 
 export default function about() {
```
Comment (Afan417) [inline pages/about.js:5]: uppercase, ``About``

**Review by Afan417 — CHANGES_REQUESTED** [review]

File: components/sections/FooterSection.js (hunk 1 above)
Comment (Fetii1) [reply components/sections/FooterSection.js:5]: deleted the whole file... I have already added the footer in the Layout.js

File: components/sections/about/AboutText.js (hunk 2 above)
Comment (Fetii1) [reply components/sections/about/AboutText.js:4]: This is in the /about page. the "AboutSection.js" is in the index.js page

File: components/layout/Navigation.js [hunk 5]
```diff
@@ -44,8 +44,8 @@ export default function Navigation() {
           >
             <ul className="flex text-base font-bold w-full p-5 flex-col md:w-auto md:p-0 md:flex-row ">
               <NavItem href="/">Ballina</NavItem>
-              <NavItem href="/about">Rreth Nesh</NavItem>
-              <NavItem href="/galeria">Galeria</NavItem>
+              <NavItem href="/About">Rreth Nesh</NavItem>
+              <NavItem href="/Gallery">Galeria</NavItem>
```
Comment (Afan417) [inline components/layout/Navigation.js:48]: ``/about /gallery``, don't uppercase

File: components/sections/AboutSection.js [hunk 6]
```diff
@@ -1,19 +1,17 @@
...
     <div className="container">
-      <div className="w-full h-auto flex my-20 justify-center">
-        <p className="text-3xl md:text-5xl px-8 pb-3 font-bold border-b-4 border-blue-950">
-          Rreth Nesh
-        </p>
-      </div>
-      <div className="flex flex-col  lg:flex-row">
-        <div className="w-full lg:text-left text-center px-10 h-auto mb-20 md:lg-0">
-          <p className="text-gray-800">
+      {/* <HeaderTitle /> */}
```
Comment (Afan417) [inline components/sections/AboutSection.js:11]: do you need this now? if no remove it

File: components/sections/AboutSection.js [hunk 7]
```diff
@@ -23,23 +21,27 @@ export default function AboutSection() {
             Deri në vitin 2014 shkolla fillore në fshatin Forinë funksiononte në kuadër të shkollës
             fillore “Përparimi” fsh. Çegran. Shkolla më 29 maj 2014 u pavarësua dhe mori emrin
             Shkolla Fillore “FORINA”.
-          </p>
-          <Button text="Lexo më shumë" />
+          </Content>
+          <Link className="py-5" href="/About">
```
Comment (Afan417) [inline components/sections/AboutSection.js:25]: ``/About,`` no, use ``/about``

**Review by Afan417 — CHANGES_REQUESTED** [review]

**Review by Afan417 — APPROVED** [review]

**Pull request merged**